src/main/java/entities/Item.java

```diff
@@ -33,7 +33,7 @@ public Item(String[] itemInfo) throws ParseException {
             if(itemInfo[5].equals("N/A")){
                 this.expirationDates = null;}
             else{
-                DateFormat dateFormat = new SimpleDateFormat("MM-dd-yyyy");
+                DateFormat dateFormat = new SimpleDateFormat("MM/dd/yyyy");
                 this.expirationDates = dateFormat.parse(itemInfo[5]);}
```

Comment:
If possible I would like to keep the original format as that is how the dates are stored currently. Please let me know if necessary to keep I can change.

Reply:
For some reason, it was throwing a format error, I'm not sure but I think the sample data had dates stored in MM/dd/yyyy format.

Reply:
I just realised, may be I hadn't updated my sample data, my bad, I'll change that!!